src/ORM/Exception/PersistenceFailedException.php

```diff
@@ -20,5 +20,5 @@
 class PersistenceFailedException extends Exception
 {
 
-    protected $_messageTemplate = 'Failed to %s an entity from %s with primary key "%s" because of: %s';
+    protected $_messageTemplate = 'Entity %s failure';
```

Comment:
We don't have to be super-official on it :) Just trying to follow it as best as possible.